### DISCUSSION TITLE
rimmod kr 만든 아조씨 전역했다는데

### DISCUSSION THREAD

**Discussion by zilrty** [RimWorldKorea/RMK#572] [일반]
rmk에 통합된걸 말해줘야 않을까

**Comment by zilrty** [comment]
https://m.dcinside.com/board/rimworld/477871

**Reply by RimWorldKorea** [reply]
이미 알고있긴 함

한창 초기에 자기것도 합쳐달라고 했었어

이제 rmk에 자진입소 시켜야지

**Reply by RimWorldKorea** [reply]
아니다 딴사람인가 하여튼 목표는 바뀌지 않는다
